src/formatDate.js

```diff
@@ -8,7 +8,65 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const YEAR_SHORT_FORMAT = 'YY';
+  const YEAR_LONG_FORMAT = 'YYYY';
+  const MONTH_FORMAT = 'MM';
+  const DAY_FORMAT = 'DD';
+  const CENTURY_21 = '20';
+  const CENTURY_20 = '19';
+  const MAX_YEAR = 30;
+  const DELIMITER = fromFormat.pop();
+  const SEPARATOR = toFormat.pop();
+  const dateParts = date.split(DELIMITER);
```

Comment:
Consider using destructuring to assign the date parts to variables with meaningful names based on the fromFormat array. This can improve readability by avoiding the need to access array elements by index.